src/main/controllers/features/limited/businessMustbeAmlRegisteredController.ts

```diff
@@ -2,15 +2,17 @@ import { NextFunction, Request, Response } from "express";
 import * as config from "../../../config";
 import { selectLang, addLangToUrl, getLocalesService, getLocaleInfo } from "../../../utils/localise";
 import { BASE_URL, LIMITED_BUSINESS_MUSTBE_AML_REGISTERED_KICKOUT, LIMITED_NAME_REGISTERED_WITH_AML, TYPE_OF_BUSINESS, AML_REGISTRATION } from "../../../types/pageURL";
+import { Session } from "@companieshouse/node-session-handler";
 
 export const get = async (req: Request, res: Response, next: NextFunction) => {
-
     const lang = selectLang(req.query.lang);
     const locales = getLocalesService();
+    const session: Session = req.session as any as Session;
+    session.setExtraData("typeOfBusinessService", "SOLE_TRADER");
```

Comment:
Check with Abhi please... I think this can be covered as his refactoring change. This data can be populated using the ACSP object..